### DISCUSSION TITLE
Native support for webp

### DISCUSSION THREAD

**Discussion by ntredwell** [linuxmint/discussions#127] [Ideas]
When will Mint provide native support for the webp format?

**Comment by mtwebster** [comment]
The `webp-pixbuf-loader` package should be installed by default in Mint 21.2. This will allow thumbnails to be generated in various programs including the file manager.

What other 'support' is needed?

**Comment by ntredwell** [comment]
I was running Mint 20. I thought I had looked to see if any changes were there, but obviously missed it. Now I am on Mint 21.2 and see all is as it should be. Sorry for raising issue unnecessarily.
